common/reducers/config.js

```diff
@@ -14,7 +15,10 @@ export type State = {
 
 export const initialState: State = {
   languageSelection: languages[0].sign,
-  nodeSelection: Object.keys(NODES)[0]
+  nodeSelection: Object.keys(NODES)[0],
+  gasPriceGwei: 21,
+  gasPriceMaxGwei: 60,
```

Comment:
if min/max values do not change during app lifetime, we probably dont want to have them in state